packages/react/src/components/SearchField/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {
+  useState,
+  useRef,
+  ChangeEvent,
+  InputHTMLAttributes,
+  ReactNode,
+  forwardRef,
+  ForwardRefRenderFunction
+} from 'react';
+import classNames from 'classnames';
+import { useId } from 'react-id-generator';
+
+import TextFieldWrapper from '../internal/TextFieldWrapper';
+import Offscreen from '../Offscreen';
+import Icon from '../Icon';
+
+interface SearchFieldProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'> {
+  label: ReactNode;
```

Comment:
This should be using `ContentNode`, see:
https://github.com/dequelabs/cauldron/blob/b7bf54c6c627676834f383c78bd31aaf9d6ace0e/packages/react/src/components/Combobox/Combobox.tsx#L36